dev-python/nilearn/nilearn-0.1.4.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 python3_4 )
+
+inherit distutils-r1
+
+MY_PV="0.1.4.post1"
+DESCRIPTION="Fast and easy statistical learning on NeuroImaging data"
+HOMEPAGE="http://nilearn.github.io/"
+SRC_URI="mirror://pypi/n/${PN}/${PN}-${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+plot"
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	"
+RDEPEND="
+	dev-python/numpy[${PYTHON_USEDEP}]
+	sci-libs/scikits_learn[${PYTHON_USEDEP}]
+	sci-libs/scipy[${PYTHON_USEDEP}]
+	sci-libs/nibabel[${PYTHON_USEDEP}]
+	plot? ( dev-python/matplotlib[${PYTHON_USEDEP}] )"
+
+S="${WORKDIR}/${PN}-${MY_PV}"
+
+src_test() {
+	emake test-code
```

Comment:
you mean nosetests? should I use a USE flag for tests? or is this somehow built-in?